packages/commerce-sdk-react/src/hooks/utils.ts

```diff
@@ -31,6 +31,10 @@ export const updateCache = (queryClient: QueryClient, cacheUpdates: CacheUpdate,
         // If an updater isn't given, fall back to just setting the data
         queryClient.setQueryData(queryKey, updater ?? data)
     )
+
+    if (cacheUpdates.clear) {
+        queryClient.clear()
```

Comment:
UPDATE: We decided to remove the "clear" functionality in favour of using "remove" since we don't want to effect any external cache since the client is potentially shared. Also, we added a namespace to all the cache keys to better avoid collisions.